source/message/OP_MSG.rst

```diff
@@ -53,6 +53,11 @@ Usage
 ``OP_MSG`` is only available in MongoDB 3.6 (``maxWireVersion >= 6``) and later.
 MongoDB drivers SHOULD perform the MongoDB handshake using ``OP_MSG`` if an API
 version was declared on the client, but MAY decide to use ``OP_QUERY``.
+However, once the driver has performed the MongoDB handshake, and it has
+inspected the value of ``maxWireVersion``, it must use either ``OP_MSG``
+(``maxWireVersion >= 6``) or ``OP_QUERY``(``maxWireVersion < 6``)
+respectively. Starting in MongoDB 6.0, using ``OP_QUERY`` for anything other
+than the MongoDB handshake will result in an error.
```

Comment:
This seems to be restating this existing rule:
> If the node supports OP_MSG, any and all messages MUST use OP_MSG, optionally compressed with OP_COMPRESSED.

What do you think about mentioning the auth commands specifically in that line instead?

Reply:
Done.